research-hub-web/src/app/services/cer-graphql.service.ts

```diff
@@ -182,18 +184,19 @@ class SubHubMap {
    * This function recursively searches through all SubHubs in SubHubMap.map for another
    * SubHub with a given slug. It returns either the parent SubHub (of type Content) or the
    * root ContentMap.map object (of type ContentMap) if the SubHub is not found.
-   * @param subHubSlug The slug o the SubHub we are looking for
+   * @param subHubSlug The slug of the SubHub we are looking for
    * @param subHub The current SubHub we are looking in
    */
-  findParentSubHub(subHubSlug: string, subHub: Content | ContentMap): Content | ContentMap {
-    if (subHub[subHubSlug]) { return subHub; }; // If the subHub is in the current SubHub
-
-    const childSubHubs: string[] = Object.keys(subHub) // Otherwise look at its children
+  findParentSubHub(subHubSlug: string, subHub: Content | ContentMap): Content | ContentMap | undefined {
```

Comment:
It might be better to throw an error, than returning `undefined`

Reply:
Leaving this for now seeing the whole cer-graphql service needs a bit of attention